workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -0,0 +1,84 @@
+import base64
+import dataclasses
+import io
+import logging
+from collections import defaultdict
+from typing import Iterable, Sequence
+
+import jinja2
+import matplotlib.pyplot as plt
+import xarray as xr
+
+
+def fig_to_b64(fig, format="png"):
+    pic_IObytes = io.BytesIO()
+    fig.savefig(pic_IObytes, format=format)
+    pic_IObytes.seek(0)
+    pic_hash = base64.b64encode(pic_IObytes.read())
+    return f"data:image/png;base64, " + pic_hash.decode()
+
+
+@dataclasses.dataclass
+class raw_html:
+    contents: str
+
+    def __repr__(self):
+        return self.contents
+
+
+template = jinja2.Template(
+    """
+<h2> {{varfilter}} </h2>
+<table>
+
+<tr>
+{% for run in runs %}
+<td><center> {{ run }} </center></td>
+{% endfor %}
+</tr>
+
+{% for varname, runs in diagnostics.items() %}
```

Comment:
```suggestion
{% for varname, runs in image_data.items() %}
```
`diagnostics` is already very overloaded and usually means `Iterable[xr.Dataset]`, can we use a different name here?